### PR TITLE
SeqProperty - clear() and reversed()

### PR DESCRIPTION
**Pull request by Starzu**

### CODE REVIEW

File: core/frontend/src/main/scala/io/udash/properties/SeqProperty.scala [hunk 1]
```diff
@@ -160,6 +164,54 @@ class TransformedReadableSeqProperty[A, B, +ElemType <: ReadableProperty[B], Ori
...
+  override def listen(l: (Seq[A]) => Any): Registration =
+    origin.listen(s => l(s.reverse))
+}
+
+class ReversedReadableSeqProperty[A](origin: ReadableSeqProperty[A, ReadableProperty[A]])
+  extends BaseReversedSeqProperty[A, ReadableProperty[A], ReadableSeqProperty[A, ReadableProperty[A]]](origin) {
+
+  override def listenStructure(l: (Patch[ReadableProperty[A]]) => Any): Registration =
+    origin.listenStructure((patch) => l(patch.copy(
+      idx = origin.size - patch.idx - patch.added.size,
```
Comment (mjjaniec) [inline core/frontend/src/main/scala/io/udash/properties/SeqProperty.scala]: shouldn't be `origin.size - patch.idx - patch.removed.size`?

File: core/frontend/src/main/scala/io/udash/properties/SeqProperty.scala [hunk 2]
```diff
@@ -160,6 +164,54 @@ class TransformedReadableSeqProperty[A, B, +ElemType <: ReadableProperty[B], Ori
...
+
+  override def set(t: Seq[A]): Unit =
+    origin.set(t.reverse)
+
+  override def replace(idx: Int, amount: Int, values: A*): Unit =
+    origin.replace(origin.size - idx, amount, values.reverse:_*)
+
+  override def listenStructure(l: (Patch[Property[A]]) => Any): Registration =
+    origin.listenStructure((patch) => l(patch.copy(
+      idx = origin.size - patch.idx - patch.added.size,
```
Comment (mjjaniec) [inline core/frontend/src/main/scala/io/udash/properties/SeqProperty.scala]: as above

**Comment by mjjaniec** [comment]
form me it's OK :)  But I'm not authorized to merge.

File: core/frontend/src/main/scala/io/udash/properties/SeqProperty.scala (hunk 1 above)
Comment (Starzu) [inline core/frontend/src/main/scala/io/udash/properties/SeqProperty.scala]: That's right. :)

**Pull request merged**